src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -53,28 +53,16 @@ const useStyles = makeStyles(({ spacing }) => ({
   },
 }));
 
-export interface ProjectListItemCardProps {
-  id: string;
-  projectImagePath: string;
-  name: string;
-  countryName: string;
-  region: string;
-  sensitivity: SensitivityType;
-  status: ProjectStatus;
-  numberOfLanguageEngagements: number;
-  esadDate: string;
-}
+export type ProjectListItemCardProps = Partial<Project> & {
+  projectImagePath?: string; // where is this comming?
```

Comment:
Projects don't currently have images, and probably won't for the initial release. We should just hard code a placeholder for now.